src/bin/bench-tps.rs

```diff
@@ -151,13 +153,14 @@ fn generate_and_send_txs(
         txs as f32 / (duration_as_s(&transfer_start.elapsed()))
     );
 
-    loop {
+    // try for 5s to get a new last_id
```

Comment:
Should we log something angry or just abort if we don’t find a new last_id?

Reply:
Yea. I will add a message and an exit(1).